nuxeo-java-client/src/main/java/org/nuxeo/client/NuxeoClient.java

```diff
@@ -230,7 +231,7 @@ protected Response request(String url, UnaryOperator<Request.Builder> method) {
             Request request = method.apply(requestBuilder).build();
             return retrofit.callFactory().newCall(request).execute();
```

Comment:
From the Javadoc of `okhttp3.Call#execute()`:

```java
   // ensure the response (and underlying response body) is closed
   try (Response response = client.newCall(request).execute()) {
     ...
   }
```

Should we use a try-with-resources statement here?